### DISCUSSION TITLE
What could be the reason, that no web control portal will be loaded?

### DISCUSSION THREAD

**Discussion by junziii** [nzymedefense/nzyme#566] [Questions]
Hello everybody!

I am really interested in this project. I started to get this working for a time but don't have got any try to work. I started always on a fresh install of raspyOS on a RaspberryPi3b. The whole described workarround was followed, but no GUI was created from the tool. I can not even find a regarded error in the log file.

Anyone does have an idea what I am doing wrong?

Do I need to create nzyme as a user or something which I have not made yet?

Thanks in advance!

junz

ll my log shows is:

pi@raspberrypi:~ $ tail -n 400 -f /var/log/nzyme/nzyme.log
14:48:26.463 [main] INFO  horse.wtf.nzyme.configuration.leader.LeaderConfigurationLoader - Skipping disabled alert callback of type [email].
14:48:47.654 [main] INFO  horse.wtf.nzyme.configuration.leader.LeaderConfigurationLoader - Skipping disabled alert callback of type [email].
14:49:08.580 [main] INFO  horse.wtf.nzyme.configuration.leader.LeaderConfigurationLoader - Skipping disabled alert callback of type [email].
14:49:29.375 [main] INFO  horse.wtf.nzyme.configuration.leader.LeaderConfigurationLoader - Skipping disabled alert callback of type [email].
14:49:49.848 [main] INFO  horse.wtf.nzyme.configuration.leader.LeaderConfigurationLoader - Skipping disabled alert callback of type [email].

**Comment by lennartkoopmann** [comment]
It looks like nzyme is restarting every 20 seconds. What does the systemd (`journalctl`) log show? Looks like there is something else going on.

**Comment by junziii** [comment]
```
-- A start job for unit network-online.target has finished successfully.
-- 
-- The job identifier is 689.
Oct 25 13:21:35 raspberrypi systemd[1]: Started Nzyme.
-- Subject: A start job for unit nzyme.service has finished successfully
-- Defined-By: systemd
-- Support: https://www.debian.org/support
-- 
-- A start job for unit nzyme.service has finished successfully.
-- 
-- The job identifier is 632.
Oct 25 13:21:35 raspberrypi sudo[854]: pam_unix(sudo:session): session closed for user root
Oct 25 13:21:38 raspberrypi sudo[873]:       pi : TTY=pts/0 ; PWD=/home/pi ; USER=root ; COMMAND=/usr/bin/systemctl status nzyme
Oct 25 13:21:38 raspberrypi sudo[873]: pam_unix(sudo:session): session opened for user root by pi(uid=0)
Oct 25 13:21:41 raspberrypi nzyme[857]: WARNING: sun.reflect.Reflection.getCallerClass is not supported. This will impact perfor
Oct 25 13:21:41 raspberrypi systemd[1]: nzyme.service: Main process exited, code=exited, status=1/FAILURE
-- Subject: Unit process exited
-- Defined-By: systemd
-- Support: https://www.debian.org/support
-- 
-- An ExecStart= process belonging to unit nzyme.service has exited.
-- 
-- The process' exit code is 'exited' and its exit status is 1.
Oct 25 13:21:41 raspberrypi systemd[1]: nzyme.service: Failed with result 'exit-code'.
-- Subject: Unit failed
-- Defined-By: systemd
-- Support: https://www.debian.org/support
-- 
-- The unit nzyme.service has entered the 'failed' state with result 'exit-code'.
Oct 25 13:21:43 raspberrypi sudo[873]: pam_unix(sudo:session): session closed for user root
```

**Reply by lennartkoopmann** [reply]
(formatted original post for readability)

It looks like nzyme is somehow aborting mid-startup. My first thought is that it might be the OOM killer kicking in. How much memory do you have available? You can find OOM messages in `dmesg`:

```
$ sudo dmesg | grep oom
```

**Comment by junziii** [comment]
I only processed the ".deb"file. What is the plan or use for the ".jar" file in the download section? Do I missed a part?

**Reply by lennartkoopmann** [reply]
The `.jar` is the Java package you'd use for a manual installation. Not required if you use the `.deb` package.

**Comment by junziii** [comment]
Okay, thanks for your effort in my Installation. Maybe you remember the words: "da ist wohl einfach der wurm drin" from Germany.. :) 

I am sure its a false acting on my side..I keep on trying, because I am even if I am not famous or rich or something often digital targeted and I am really happy to have good hope for a running "guardian" which you have created with nzyne! Thanks a lot for this!

Short before my post I found a nice install "mesh-up" where 2 configurations were spoken about. Maybe the hyperlink of this Video is something to add to the wiki if it is not already there (https://www.youtube.com/watch?v=XFi-cJZSY5M).

Just want to give the positive input, that the "reaching the GUI" could easy be approved by some more detailed information about its address and the related port. In addition to that, for me as an non professional but advanced user, I felt in case of the config file, a little bit of "alone" so more comments would leave a "warmer" feeling for new user I guess.. For example: It works without x and y configured from the start... or the definition from which ip-range nzyme will accept connections.

One more thing, the possible influence of the "8 hours" to get certificate topic is not at all clear for me as not high skilled user...

Thanks for support and this really interesting tool!

Junz

**Comment by shandshellin** [comment]
https://github.com/lennartkoopmann/nzyme/discussions/657
